src/lib/assets/yaml.ts

```diff
@@ -11,6 +11,118 @@ import { deployment, moduleSecret, namespace, watcher } from "./pods";
 import { clusterRole, clusterRoleBinding, serviceAccount, storeRole, storeRoleBinding } from "./rbac";
 import { webhookConfig } from "./webhooks";
 
+// Overrides file generated from assets
```

Comment:
```suggestion
// Helm Chart overrides file (values.yaml) generated from assets
```
Embarrassingly, it took me a while to figure out that "overrides" meant "values.yaml" in this context.  Would you mind if we just popped "values.yaml" into the comment somewhere?  It'd have helped me get the point right away.

As a bonus, it'd provide a place for "people who see the output file & wanna understand the source that made it" land in the right place.

Reply:
great idea